src/main/resources/com/puppycrawl/tools/checkstyle/messages.properties

```diff
@@ -19,3 +19,4 @@ PackageObjectFactory.unableToInstantiateExceptionMessage=Unable to instantiate '
    Please recheck that class name is specified as canonical name or read how to configure \
    short name usage https://checkstyle.org/config.html\#Packages. Please also \
    recheck that provided ClassLoader to Checker is configured correctly.
+parse.exception=Java specific modules are skipped due to a parse exception - {0}
```

Comment:
```suggestion
parse.exception= Java specific (TreeWalker-based) modules are skipped due to an exception during parsing - {0}
```

Reply:
We should keep some tie to TreeWalker in the message to make it clear that it happened in some child of TreeWalker. Also, I would prefer “during parsing” since this could also be an exception thrown in the JavaAstVisitor, right?

Reply:
we don't add a space after `=` for `.properties` file I guess we should leave it as it is for consistency

Reply:
Extra space was typo

Reply:
I am good.
I just need keep few words:  Java, Parse, Exception . To let users easily match it to skipFileOnJavaParseException